config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics-main.yaml

```diff
@@ -1,20 +1,20 @@
 periodics:
 - name: periodic-cluster-api-provider-azure-conformance-main
+  cluster: k8s-infra-prow-build
```

Comment:
TODO: See if we can get this on an AKS cluster in the future to avoid cross cloud bandwidth and releive pressure on GCP budget. 

If not, we can at least look into putting it on the EKS cluster as a fallback, we have more room on that budget and this isn't inherently tied to GCP.

Reply:
(ref: https://kubernetes.slack.com/archives/CCK68P2Q2/p1721258915827939?thread_ts=1720991218.482619&cid=CCK68P2Q2)

Reply:
agreed, we have been focusing on getting things moved over and after the deadline can revisit getting PROW running on the Azure subscription

Reply:
Thanks! I just ask if we get stuck on that part that we look at copying the secrets to EKS and run there, as GCP is at ~86.5% spend rate vs budget and the others are not nearly that high, while we still have more things to migrate that will probably be harder to move.

Does NOT need to happen in this PR, but I want to make sure that's on our near-term backlog if possible. Thank you!

Reply:
> If not, we can at least look into putting it on the EKS cluster as a fallback, we have more room on that budget and this isn't inherently tied to GCP.

we have a todo to add EKS federated credentials https://github.com/kubernetes/k8s.io/pull/6925/files#diff-cf32112272ec05a893a46f5e2c8430000939902241ac18e1326414cb04db13dcR15.  We just need to know the EKS `issuer`

Reply:
This is just the first wave for the migration to make sure things are working.  We have a bunch more jobs to move over after this. How should we split up the rest of the jobs we move? Should they all go over to EKS?

Reply:
> How should we split up the rest of the jobs we move? Should they all go over to EKS?

Currently: Yes, if possible. The clusters should auto-scale, and AWS spend is at more like 67% (with the "same" credit budget even through cloud credits are somewhat apple:orange)

We can rebalance later, and in most cases we have confidence that jobs on the AWS cluster could run anywhere, whereas some of the older workloads on GCP may be using other GCP specific things like cloud-build, artifact registry / GCR, etc. for aspects that are not pure-kubernetes.